src/components/TokenBottomSheet.test.tsx

```diff
@@ -201,7 +184,7 @@ describe('TokenBottomSheet', () => {
 
     expect(queryByTestId('cUSDTouchable')).toBeNull()
     expect(queryByTestId('cEURTouchable')).toBeNull()
-    expect(queryByTestId('TTTouchable')).toBeTruthy()
+    expect(getByTestId('TTTouchable')).toBeTruthy()
```

Comment:
😅 not really necessary of course but i am making these changes where i see them because of [this](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence) - the error messages are more helpful with the `getBy` selectors if they fail, the error will also print the tree that is rendered which has helped me before

Reply:
oh I see, yes `getBy` is better for this, 
The only small drawback is the code looks less consistent when we're checking both for existence and absence.